dashboard_data_writers/metrics_tables.py

```diff
@@ -0,0 +1,127 @@
+           cache_args=['start_time', 'end_time', 'variable', 'truth', 'metric',
+                       'time_grouping', 'grid', 'mask', 'region'],
+           cache=True)
+def summary_metrics_table(start_time, end_time, variable,
+                          truth, metric, time_grouping=None,
+                          grid='global1_5', mask='lsm', region='global'):
+    """Runs summary metric repeatedly for all forecasts and creates a pandas table out of them."""
+    forecasts = ['fuxi', 'salient', 'ecmwf_ifs_er', 'ecmwf_ifs_er_debiased', 'climatology_2015',
+                 'climatology_trend_2015', 'climatology_rolling']
+    leads = ["week1", "week2", "week3", "week4", "week5", "week6"]
+    df = _summary_metrics_table(start_time, end_time, variable, truth, metric, leads, forecasts,
+                                time_grouping=time_grouping,
+                                grid=grid, mask=mask, region=region)
+
+    print(df)
+    return df
+
+
+@dask_remote
+@cacheable(data_type='tabular',
+           cache_args=['start_time', 'end_time', 'variable', 'truth', 'metric',
+                       'time_grouping', 'grid', 'mask', 'region'],
+           cache=True)
+def station_metrics_table(start_time, end_time, variable,
+                          truth, metric, time_grouping=None,
+                          grid='global1_5', mask='lsm', region='global'):
+    """Runs summary metric repeatedly for all forecasts and creates a pandas table out of them."""
+    forecasts = ['era5', 'chirps', 'imerg']
+    leads = ["daily", "weekly", "biweekly", "monthly"]
+    df = _summary_metrics_table(start_time, end_time, variable, truth, metric, leads, forecasts,
+                                time_grouping=time_grouping,
+                                grid=grid, mask=mask, region=region)
+
+    print(df)
+    return df
+
+
+@dask_remote
+@cacheable(data_type='tabular',
+           cache_args=['start_time', 'end_time', 'variable', 'truth', 'metric',
+                       'time_grouping', 'grid', 'mask', 'region'],
+           cache=True)
+def biweekly_summary_metrics_table(start_time, end_time, variable,
+                                   truth, metric, time_grouping=None,
+                                   grid='global1_5', mask='lsm', region='global'):
+    """Runs summary metric repeatedly for all forecasts and creates a pandas table out of them."""
+    forecasts = ['perpp', 'ecmwf_ifs_er', 'ecmwf_ifs_er_debiased', 'climatology_2015',
+                 'climatology_trend_2015', 'climatology_rolling']
+    leads = ["weeks34", "weeks56"]
+    df = _summary_metrics_table(start_time, end_time, variable, truth, metric, leads, forecasts,
+                                time_grouping=time_grouping,
+                                grid=grid, mask=mask, region=region)
+
+    print(df)
+    return df
+
+
+__all__ = ['summary_metrics_table', 'biweekly_summary_metrics_table', 'station_metrics_table']
+
+if __name__ == "__main__":
```

Comment:
TODO: @adkinsjd , maybe we can update this with how you update the tables? Is that useful?